{
    "config": {
        "abort": {
            "already_configured": "Cas\u00e9ta bridge already configured.",
            "cannot_connect": "Cancelled setup of Cas\u00e9ta bridge due to connection failure."
        },
        "error": {
            "cannot_connect": "Failed to connect to Cas\u00e9ta bridge; check your host and certificate configuration."
        },
        "step": {
            "user": {
                "data": {
                    "ca_certs": "Filename of bridge CA file (caseta-bridge.crt) generated by get_lutron_cert.py",
                    "certfile": "Filename of client certificate file (caseta.crt) generated by get_lutron_cert.py",
                    "host": "Cas\u00e9ta bridge IP address",
                    "keyfile": "Filename of keyfile (caseta.key) generated by get_lutron_cert.py"
                },
                "description": "Please enter the configuration information to connect to the Cas\u00e9ta bridge; use get_lutron_cert.py to generate the necessary certificate files.",
                "title": "Configure Cas\u00e9ta bridge"
            }
        }
    }
}